src/Performance/Converters/RiskStatisticsConverter.ts

```diff
@@ -0,0 +1,108 @@
+/* *
+ *
+ *  (c) 2009-2025 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Mateusz Bernacik
+ *
+ * */
+
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import MorningstarConverter from '../../Shared/MorningstarConverter';
+import type { PerformanceConverterOptions } from '../PerformanceOptions';
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+
+export class RiskStatisticsConverter extends MorningstarConverter {
+
+
+    /* *
+     *
+     *  Constructor
+     *
+     * */
+
+
+    public constructor (
+        options?: PerformanceConverterOptions
+    ) {
+        super(options);
+    }
+
+
+    /* *
+     *
+     *  Functions
+     *
+     * */
+
+
+    public override parse (
+        options: PerformanceConverterOptions,
+        hasMultiple?: boolean
+    ): void {
+        const table = this.table,
+            columnSuffix = hasMultiple ? `_${options.json.PortfolioName}` : '',
+            riskStatistics = options.json.Risks.RiskStatistics;
+
+        if (riskStatistics.length) {
+            riskStatistics.forEach((statistic, i) => {
+                const {
+                    Benchmark,
+                    Portfolio,
+                    Security,
+                    TrailingTimePeriod,
+                    DataFrequency
+                } = statistic;
+
+                table.setCell('TrailingTimePeriod', i, TrailingTimePeriod);
+                table.setCell('DataFrequency', i, DataFrequency);
+
+                for (const key of Object.keys(Benchmark) as Array<keyof typeof Benchmark>) {
```

Comment:
Q: Can `benchmark` be undefined?

Reply:
@pawelfus I don't think so. Even if benchmark is not included in the body it still returns benchmark.